### ISSUE TITLE
Application stays running after being closed

### ISSUE THREAD

**Issue by skycatminepokie** [Zodsmar/SeedSearcherStandaloneTool#119]
Closing the application window doesn't stop the application. You can't move the .jar file because java still has it open. Why's that?

**Comment by Zodsmar** [comment]
Um, I have no idea I have never experienced this or seen any comments on it. The new version for 1.18 is being completely rewritten from the ground up so this shouldn't be a problem.

**Comment by Zodsmar** [comment]
Yeah I am reopening this because while doing the rewrite I was testing against the old application and I noticed actually lingering as well. Need to find the source of that so it doesnt happen in the rewrite. Leaving open till I can verify its fixed
